from-dump/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM quetzacoalt/docker-internal-images:1.7
+
+ENV PS_DOMAIN localhost
+ENV DUMP_PHP_FILES prestashop-dataset
+ENV DUMP_SQL_FILE prestashop-dataset.sql
+
+# Copy files
+COPY $DUMP_PHP_FILES/ /var/www/html/
+RUN ln -s /var/www/html/ /var/www/html/$DUMP_PHP_FILES/
```

Comment:
Hello,
Yes, it's a structure issue from a real shop installed in /var/www/html but i don't know if it's important.
Just, you have in ps_shop_url localhost & prestashop-dataset in domain & path.
That all i can say